lib/git_reflow/commands/setup.rb

```diff
@@ -25,5 +25,12 @@
     GitReflow::Config.add "constants.minimumApprovals", ask("Set the minimum number of approvals (leaving blank will require approval from all commenters): "), local: reflow_options[:project_only]
     GitReflow::Config.add "constants.approvalRegex", GitReflow::GitServer::PullRequest::DEFAULT_APPROVAL_REGEX, local: reflow_options[:project_only]
     
+    say("Enter the follow information associated with your remote repository: (The information is used to run 'git reflow deliver'.")
```

Comment:
In order to use the github_api gem, we need to provide the payload, which needs to owner and repository name as well as the auth-token in order to make the HTTP request. I decided to put these in the git config file and prompt users to enter these.